dts/bindings/display/backlight-pwm.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024, Maritn Kiepfer
+# SPDX-License-Identifier: Apache-2.0
+
+description: Backlight PWM node
+
+compatible: "backlight-pwm"
```

Comment:
probably use zephyr as vendor? `zephyr, backlight-pwm`

Reply:
no, we should not do that here, there is a linux compatible we should match